docs/DeveloperGuide.md

```diff
@@ -901,30 +901,275 @@ testers are expected to do more *exploratory* testing.
+   
+   3. Other incorrect inputs: `meeting 0 mt/19-10-2022-10:34`, `meeting 2 mt/31-09-2022-10:34`, `meeting 2 mt/19-10-2022-10:60`
+
+2. Re-adding the same meeting to the same person
+
+   1. Use `meeting 2 mt/19-10-2022-10:34` to add a meeting.
+   
+   2. Re-enter `meeting 2 mt/19-10-2022-10:34`<br>
+      Expected: Success message shown in command display.
+      
+      1. Click on the second client on the `result display`.<br>
+         Expected: Only one `19 October 2022 10:34`. No duplicate meetings.
+      
 ### Deleting a person
 
 1. Deleting a person while all persons are being shown
 
    1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
 
    1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the command display.
+
+   1. Test case: `delete 0`<br>
+      Expected: No person is deleted. Error details shown in the command display.
+      
+   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+      Expected: Similar to previous.
+      
+1. Deleting a person while a filtered list of persons is being shown
+
+   1. Prerequisites: Find certain persons using the `find` command. Multiple persons in the list.
+
+   1. Test case: `delete 1`<br>
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the command display.
+      
+      1. Use the `list` command to show all persons.
+         Expected: The same contact is deleted from the main list.
 
    1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No person is deleted. Error details shown in the command display.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
 
-### Saving data
+### Remove past meetings
 
-1. Dealing with missing/corrupted data files
+1. Remove past meetings
+   1. Add multiple meeting to multiple clients in FABook with meeting times after the present time.
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   1. Add multiple meeting to multiple clients in FABook with meeting times before the present time.
+   
+   3. Enter `sync`.<br>
+      Expected: Meetings added in step 2 are removed. Success message displayed shown in the command display.
+      
+### Saving data
 
-2. _{ more test cases …​ }_
+1. Dealing with missing data files
+
+   1. Launch the app and exit after any number of actions.
+   
+   3. In the same directory as the app, find folder `data`.
```

Comment:
wrong numbering

Reply:
Lol i decided to give up on the numbering 😭 Apparently it will be correct when viewed in rich text.